src/api/python/_api.py

```
@@ -925,22 +937,20 @@ def get_xfer_descs(
     ) -> nixlBind.nixlXferDList:
         # can add check for DLPack input
 
+        descs_len = None
+        with contextlib.suppress(TypeError):
+            descs_len = len(descs)
+
         if isinstance(descs, nixlBind.nixlXferDList):
             return descs
         elif isinstance(descs, nixlBind.nixlRegDList):
```

Ruff (0.15.10)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)